lib/libraries/rimraf.js

```diff
@@ -0,0 +1,360 @@
+const assert = require("assert")
```

Comment:
Why do we need to include the rimraf library here if we are using the dependency in package.json?